onebusaway-android/src/main/java/org/onebusaway/android/ui/ArrivalsListFragment.java

```diff
@@ -982,11 +982,19 @@ private void doSortBy() {
             Toast.makeText(getActivity(), r.getString(R.string.stop_info_option_sort_route),
                     Toast.LENGTH_SHORT).show();
             newValue = 1;
+            //Analytics
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
+                    getActivity().getString(R.string.analytics_action_button_press),
+                    getActivity().getString(R.string.analytics_label_sort_by_eta_arrival));
```

Comment:
I believe this should be `analytics_label_sort_by_route_arrival` - it should match up with the `Toast` message.